src/relayers/relayersDataService.ts

```diff
@@ -99,7 +98,7 @@ export async function getAllRelayers(
                 ) reg
                 LEFT OUTER JOIN
                   (
-                    SELECT relayer_id, sum(delta::BIGINT) as lifetime_sla_change
+                    SELECT vault_id as relayer_id, sum(delta::BIGINT) as lifetime_sla_change
```

Comment:
I know I just merged this, but should that be `vault_id`?

Reply:
The view definition defines the column as vault_id (due to erroneous copy-pasting at some point in the past, as the vault and relayer views are very similar). The current error 500 is (partially) caused by the query asking for relayer_id when that column doesn't exist.

Yes, it's absolutely terrible, but the query will work like this. There are a few aspects of the database that could benefit from cleaning up, but I didn't want to risk renaming the column right now when we're trying to get things working before tomorrow.